src/app/pages/components/tree/input.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, OnInit } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+    selector: 'ngx-modal',
+    templateUrl: './input.component.html',
+})
+export class InputComponent implements OnInit {
+
+    public form: FormGroup;
```

Comment:
The public is implicit, and is never used in this code base (at least not that I saw)